tasks/12_title-case/title-case.js

```diff
@@ -0,0 +1,7 @@
+function titleCase(str) {
+  let arr = str.split(' ');
+  arr = arr.map((item) =>
```

Comment:
It is a bad practice to rewrite array variable, instead, you could return a new array, produced by the map and joined.